webpack.config.js

```diff
@@ -42,6 +45,12 @@ module.exports = {
           'postcss-loader',
           'sass-loader?sourceMap',
         ],
+      }, {
+        test: /\.css$/,
+        loaders: [
+          'style-loader',
+          'css-loader',
```

Comment:
yeah .. of course, I want to do that, but i can't.

I tried to import `.scss` for grommet, but it will fail when it complier.

so for now, I use this.
`import '../node_modules/grommet/grommet.min.css';`